currency/storage.go

```diff
@@ -581,7 +581,7 @@ func (s *Storage) SeedForeignExchangeRates() error {
 	return s.updateExchangeRates(rates)
 }
 
-// UpdateForeignExchangeRates sets exchange rates on the FX map
+// updateExchangeRates sets exchange rates on the FX map
```

Comment:
🐒 Nitpick: The rest of the comment isn't particularly clear, and since this is now both well named and private, we don't need a comment.
```suggestion
```